lib/components/molecules/StickyHeader/types/index.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+import type, { Node } from 'react';
+
+export type Props = {
```

Comment:
Your component works with a state, isSticky we should define that in types too

Reply:
Fixed.